tool/pom.xml

```diff
@@ -1,133 +1,78 @@
-        </mailingList>
-    </mailingLists>
-
-    <issueManagement>
-        <system>GitHub Issues</system>
-        <url>https://github.com/antlr/antlr4/issues</url>
-    </issueManagement>
-
-    <scm>
-        <url>https://github.com/antlr/antlr4/tree/master</url>
-        <connection>scm:git:git://github.com/antlr/antlr4.git</connection>
-        <developerConnection>scm:git:git@github.com:antlr/antlr4.git</developerConnection>
-        <tag>HEAD</tag>
-    </scm>
-
-    <distributionManagement>
-        <snapshotRepository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </snapshotRepository>
-        <repository>
-            <id>ossrh</id>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
-
-
-    <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.11</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
-            <version>2.44.0</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>jetty-server</artifactId>
-            <version>8.1.16.v20140903</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.antlr</groupId>
-            <artifactId>antlr4-runtime</artifactId>
-            <version>4.5</version>
-        </dependency>
-        <dependency>
-            <groupId>org.antlr</groupId>
-            <artifactId>antlr-runtime</artifactId>
-            <version>3.5.2</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.antlr</groupId>
-            <artifactId>ST4</artifactId>
-            <version>4.0.8</version>
-            <scope>compile</scope>
-        </dependency>
-    </dependencies>
-
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.antlr</groupId>
+    <artifactId>antlr4-master</artifactId>
+    <version>4.5.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>antlr4</artifactId>
+  <name>ANTLR 4 Tool</name>
+  <description>The ANTLR 4 grammar compiler.</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
```

Comment:
Have you tried running the javascript browser tests?

> Le 21 juin 2015 à 04:04, Sam Harwell <notifications@github.com> a écrit :
> 
> In tool/pom.xml <https://github.com/antlr/antlr4/pull/920#discussion_r32887500>:
> 
> > +    <artifactId>antlr4-master</artifactId>
> > +    <version>4.5.1-SNAPSHOT</version>
> > +  </parent>
> > +  <artifactId>antlr4</artifactId>
> > +  <name>ANTLR 4 Tool</name>
> > +  <description>The ANTLR 4 grammar compiler.</description>
> > +
> > +  <dependencies>
> > +    <dependency>
> > +      <groupId>junit</groupId>
> > +      <artifactId>junit</artifactId>
> > +      <version>4.11</version>
> > +      <scope>test</scope>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.seleniumhq.selenium</groupId>
> @ericvergnaud <https://github.com/ericvergnaud> I looked through the artifact extensively to make sure the dependency really isn't required. It sounds like you're running into an issue with another artifact which happens to be resolved by including this dependency. This would indicate that the missing dependency errors are actually a bug in the other module, not this one.
> 
> To date, I haven't been able to reproduce an ANTLR environment where omitting this dependency causes failures. If you have an example, I would certainly be willing to help narrow down this issue with the hope of solving it while also not requiring ANTLR/Java developers to download this large dependency.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/920/files#r32887500>.
>